src/views/Generator/RecordingSelector.tsx

```diff
@@ -65,7 +65,10 @@ export function RecordingSelector() {
             id="recording-selector"
             placeholder="Select recording"
             css={css`
-              width: 250px;
+              width: 200px;
```

Comment:
Now that we have more horizontal space, I'd expect it to be wider rather than narrower than before, because with the addition of the starting URL in recording names, the names became even longer.

Reply:
I was mainly trying to keep things concise on the right side of the top bar, but given that names now can be longer, it makes sense to make it wider.